packages/devtools/test/utils_test.dart

```diff
@@ -127,24 +127,84 @@ void main() {
     });
 
     test('longestFittingSubstring', () {
-      const str = 'ComponentElement.performRebuild';
+      const asciiStr = 'ComponentElement.performRebuild';
+      const nonAsciiStr = 'ԪԪ';
+      void slowMeasureCallback(_) => 100;
```

Comment:
This type is wrong. You should get an analysis error once you add types elsewhere.

Reply:
done.